app/controllers/organizations_controller.rb

```diff
@@ -18,6 +18,7 @@ def index
   end
 
   def show
+  @movements = @organization.account.movements.order("created_at DESC").page(params[:page]).per(10)
```

Comment:
Este es el bueno  :+1: 
Movement Load (160.6ms)  SELECT  "movements".* FROM "movements" WHERE "movements"."account_id" = $1  ORDER BY created_at DESC LIMIT 10 OFFSET 0  [["account_id", 2]]
  Movement Load (160.6ms)  SELECT  "movements".* FROM "movements" WHERE "movements"."account_id" = $1  ORDER BY created_at DESC LIMIT 10 OFFSET 0  [["account_id", 2]]
=> EXPLAIN for: SELECT  "movements".* FROM "movements" WHERE "movements"."account_id" = $1  ORDER BY created_at DESC LIMIT 10 OFFSET 0 [["account_id", 2]]
                                               QUERY PLAN
--------------------------------------------------------------------------------------------------------
 Limit  (cost=14.45..14.47 rows=7 width=32)
   ->  Sort  (cost=14.45..14.47 rows=7 width=32)
         Sort Key: created_at
         ->  Bitmap Heap Scan on movements  (cost=4.21..14.35 rows=7 width=32)
               Recheck Cond: (account_id = 2)
               ->  Bitmap Index Scan on index_movements_on_account_id  (cost=0.00..4.21 rows=7 width=0)
                     Index Cond: (account_id = 2)
(7 rows)